src/bun.js/bindings/BunProcess.cpp

```diff
@@ -77,7 +77,9 @@ namespace Bun {
 
 using namespace JSC;
 
-#define REPORTED_NODE_VERSION "22.2.0"
+#ifndef REPORTED_NODEJS_VERSION
+#define REPORTED_NODEJS_VERSION "22.3.0"
```

Comment:
if the build system is going to define this, we shouldnt need a fallback here. these lines can be removed entirely.

it feels weird to have a (ideally unused) fallback value.

similar comment can be said about the build.zig but that default is more tolerable because `zig build check` is convenient to have no arguments on.